yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
i don't think this file should be here ✂️

Reply:
Can you help me understand how this would have gotten into the commit if it's not supposed to be there? I didn't add it explicitly. 

I'm asking because I didn't do it on purpose, which means it's likely to happen again unless I know how to stop it.

Reply:
the only thing I can think of is if you were to inadvertently run `yarn` from the project root thinking you were actually in `ember-flight-icons`

Reply:
that makes sense, ty!